spec/factories/conferences.rb

```diff
@@ -1,11 +1,12 @@
 FactoryGirl.define do
-  factory :conference do
-    name { [FFaker::CheesyLingo.title, 'Conf'].join ' ' }
-    tickets 2
+    factory :conference do
+      name [FFaker::CheesyLingo.title, 'Conf'].join(' ')
+      starts_on { Time.utc(Date.today.year, 7, 7) }
```

Comment:
I removed the tickets, because they are not a requirement for validation. Plus, the only test that refers to the ticket attributes, overrides it.